concent_api/core/tests/test_core_views.py

```diff
@@ -218,8 +221,7 @@ def test_send_should_return_http_400_if_data_is_incorrect(self):
             ),
             content_type                        = 'application/octet-stream',
         )
-        self.assertEqual(response.status_code, 400)
-        self.assertIn('error', response.json().keys())
+        self._test_400_response(response)
```

Comment:
Why here is no error code?

Reply:
No, because here the response is received directly from decorator handling the errors.